### PR TITLE
Mixture model of frame posteriors

### PR DESCRIPTION
**Pull request by teinhonglo**
The script which combines the outputs of two (or more) different neural networks and then 
use the combined posterior probabilities for decoding.

In our experiments, Improvement in WER could be 0.5 ~ 1.5.

### CODE REVIEW

**Comment by teinhonglo** [comment]
Dear Povey,

Thanks for your advice.
I have already modify my code based on your advice.
1. rename "score_fusion.sh" to "decode_score_fusion.sh".
2. checking that they have the same frame-subsampling-factor and tree.

Best Regards,
Tien-Hong

File: egs/wsj/s5/steps/nnet3/decode_score_fusion.sh [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+[ $num_threads -gt 1 ] && thread_string="-parallel --num-threads=$num_threads"
+
+mkdir -p $dir/temp
+
+for i in `seq 0 $[num_sys-1]`; do
+  srcdir=${model_dirs[$i]}
+  
+  model=$srcdir/$iter.mdl
+  if [ ! -f $srcdir/$iter.mdl ]; then
+    echo "$0: ERROR: no such file $srcdir/$iter.raw. Trying $srcdir/$iter.mdl exit" && exit 1;
```
Comment (danpovey) [inline egs/wsj/s5/steps/nnet3/decode_score_fusion.sh]: Please say Error, not ERROR.

**Comment by teinhonglo** [comment]
Is there anything I could or should do?

**Comment by danpovey** [comment]
merged.  Thanks!

**Pull request merged**